docs/UserGuide.md

```diff
@@ -4,339 +4,1044 @@
+This ensures that as you apply to more roles and receive updates on your applications, every piece of information is 
+systematically stored, making tracking and follow-ups more manageable.
+
+<box type="tip" light>
+
+* Regularly adding applications immediately after submission ensures you don't miss out on tracking any application.
+* Remember to update the `STATUS` of your applications as they progress. This way, you'll always be aware of which stage each application is at.
+
+</box>
+
+__________________________________________________________________________________________________________________
 
 ### Editing an internship application : `edit`
 
-Edits an internship application in the internship application list.
+The `edit` command offers flexibility in managing your internship applications. 
+
+As the status of your applications change, or if you notice inaccuracies in the details you've entered, 
+the `edit` command ensures you can **update any part of your application** without any hassle.
+
+#### Command Usage
+
+**Command**:
+```
+edit INDEX [n/COMPANY_NAME] [r/ROLE] [c/CYCLE] [s/STATUS] [d/DEADLINE]
+```
+
+#### Parameters:
+
+<box type="important" seamless>
+
+At least **one** of the optional parameters must be provided.
+</box>
+
+1. **INDEX**: Represents the position of the internship application in your list that you wish to update.
+   <box type="info" seamless>
+
+   The `INDEX` should be a positive integer. For example: `1`, `2`, `3`, etc. This corresponds to the application's position in your list.
+   </box>
+
+2. **n/COMPANY_NAME** (Optional): The updated company name.
+3. **r/ROLE** (Optional): The updated internship role.
+4. **c/CYCLE** (Optional): The updated internship cycle.
+5. **s/STATUS** (Optional): The updated status of your application.
+6. **d/DEADLINE** (Optional): The updated deadline or any related dates.
 
-**Format**: `edit INDEX [n/COMPANY_NAME] [r/ROLE] [c/CYCLE] [s/STATUS] [d/DEADLINE]`
+#### Example:
 
-**Parameters**:
-* `INDEX`: Index of internship application entry to be edited
-    * `INDEX` must be a positive integer, such as, `1`, `2`, `3`, etc.
-* At least one of the optional fields must be provided.
-    * Existing values will be updated to the input values.
+Let's say you want to edit the second internship application in your list to change the company name to **Amazon**, 
+update the role to **Data Analyst Intern**, and modify the status to **Interview**. The command would look like this:
 
-**Examples**:
-* `edit 1 s/Applied` edits the status of the first internship application entry (in the list) as `Applied`
+```
+edit 2 n/Amazon r/Data Analyst Intern s/Interview
+```
+
+**_UPDATE PICTURE HERE_**
 
-**Expected Output**:
 ![edit command example](images/editExample.jpg)
 
-### Adding a note to a specific internship : `note`
+You can see how the original details of the application are updated after executing the `edit` command. 
```

Comment:
I think there have been updated note commands with the addition of the ability to store multiple notes

Reply:
Will check with You Wei if he has provided these!